build.gradle

```diff
@@ -30,7 +30,7 @@ apply plugin: "net.minecraftforge.gradle.launch4j"
 minecraft.version = "1.12.2"
 
 minecraft {
-    mappings = 'snapshot_nodoc_20171003'
+    mappings = 'snapshot_nodoc_20171004'
```

Comment:
Please don't change the mapping. It took me a while to test the current existing mapping and, although I doubt much has changed in one day, I don't want to have to test it again.